Userland/Libraries/LibSoftGPU/Clipper.h

```diff
@@ -15,7 +15,7 @@ namespace SoftGPU {
 
 class Clipper final {
 public:
-    enum class ClipPlane : u8 {
+    enum class ClipFrustrum : u8 {
```

Comment:
I don't get this name change - apart from the fact that it contains a typo ("Frustrum" instead of "Frustum"), these represent planes and not frustums:
https://en.wikipedia.org/wiki/Frustum